src/features/footer/MobileFooter.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { sendTrackEvent } from '@edx/frontend-platform/analytics';
+import { ensureConfig } from '@edx/frontend-platform/config';
+import { AppContext } from '@edx/frontend-platform/react';
+
+ensureConfig([
+  'ADA_URL',
+  'LMS_BASE_URL',
+  'LOGO_TRADEMARK_URL',
+  'PRIVACY_POLICY_URL',
+  'TERMS_OF_SERVICE_URL',
+  'COPY_RIGHT_STRING',
+], 'DesktopFooter component');
+
+const EVENT_NAMES = {
+  ADA_LINK: 'edx.ada.footer.link',
+  PRIVACY_POLICY_LINK: 'edx.privacy.policy.footer.link',
+  TERMS_OF_SERVICE_LINK: 'edx.terms.of.service.footer.link',
+};
+
+const copySymbol = String.fromCodePoint(0x00A9);
+
+class MobileFooter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.externalLinkClickHandler = this.externalLinkClickHandler.bind(this);
+  }
+
+  externalLinkClickHandler(event) {
+    const label = event.currentTarget.getAttribute('href');
+    const eventName = EVENT_NAMES.ADA_LINK;
```

Comment:
Shouldn't this change depending on which link was clicked?

Reply:
Yes I will update with the cases